### PR TITLE
Subscription flow improvement

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Update the subscription flow
New screen for Application Registration
New screen for Credential list

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | ab0da93 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-501.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop can you start reviewing my english?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 1]
```diff
@@ -330,9 +330,9 @@ When subscribing to the plan, you will be able to see the emails and billing add
 
 ![Subscription Request screen](/Images/central/integrate_with_central/SubscriptionRequestDefinition.png)
 
-## Customize access request screen
+## Customize Application Registration screen
 
-To customize the access request screen, you need an `AccessRequestDefinition`. This object will contain the screen definition of the information required to provision access to a service and the output the provider wants to return to the consumer. This object is scoped per environment, meaning that if you have multiple environments, you must duplicate the access request definition for each individual environment.
+To customize the Application Registration screen, you need an `AccessRequestDefinition`. This object will contain the screen definition of the information required to provision access to a service and the output the provider wants to return to the consumer. This object is scoped per environment, meaning that if you have multiple environments, you must duplicate the Application Registration definition for each individual environment.
```
Comment (georgiana-caras) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:335]: you must duplicate the AccessRequestDefinition for each individual environment.

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 2]
```diff
@@ -471,17 +471,17 @@ Sample of an APIServiceInstance (json format) using the previous accessRequestDe
 }
 ```
 
-If the APIService instance is already linked to an asset prior to adding the access request definition, you must create a new release of the asset and a new release of the product to take the modification into consideration in the Marketplace.
+If the APIService instance is already linked to an asset prior to adding the Application Registration definition, you must create a new release of the asset and a new release of the product to take the modification into consideration in the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:474]: If the APIService instance is already linked to an asset prior to adding the AccessRequestDefinition object

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by georgiana-caras — APPROVED** [review]

**Pull request merged**